custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -63,11 +63,13 @@ def person(self):
                 # 'owner_id': self._location.location_id,
                 'update': {
                     'aadhaar_number': self.patient_detail.paadharno,
+                    'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
                     'contact_phone_number': validate_number(self.patient_detail.pmob),
                     'current_address': self.patient_detail.paddress,
                     'current_address_district_choice': self.patient_detail.Dtocode,
                     'current_address_state_choice': self.patient_detail.scode,
+                    'dob_known': False,
```

Comment:
Also, I think when this is false, we still calculate a DOB in the app, so would be good to do that here too. (for [eg check this case](https://india.commcarehq.org/a/enikshay-test/reports/case_data/45953ec6-304a-4c30-984a-05e6840dfde9/).)